panda/src/audio/audioSound.cxx

```diff
@@ -196,3 +196,58 @@ operator << (ostream &out, AudioSound::SoundStatus status) {
 
   return out << "**invalid AudioSound::SoundStatus(" << (int)status << ")**";
 }
+
+/**
+ * Get the comment attached to this AudioSound as a list of strings.
+ */
+std::vector<std::string> AudioSound::
```

Comment:
Let's use the `vector_string` class everywhere, this is an alias defined in `vector_string.h` and uses Panda allocators

Reply:
Fixed. Thank you for the assistance 😊 *(no I am not an ai network if it reads like that)*